omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ConceptResource1_8.java

```diff
@@ -462,9 +462,14 @@ public static List<ConceptMap> getMappings(Concept instance) {
 	 * @param instance the delegate instance to get the display name off
 	 */
 	@PropertyGetter("display")
-	public String getDisplayName(Concept instance) {
-		ConceptName cn = instance.getName();
-		return cn == null ? null : cn.getName();
+	public String getDisplayString(Concept instance) {
+		String localization = getLocalization("Concept", instance.getUuid());
+		if (StringUtils.isNotBlank(localization)) {
+			return localization;
+		} else {
+			ConceptName cn = instance.getName();
+			return cn == null ? null : cn.getName();
```

Comment:
I realize this is pre-existing behavior not related to this PR, but we should never return null from this method, there should always be something.  The "getDisplayString" method on Concept (at least in later version of OpenMRS) basically has this same logic, but it returns Concept.toString() as a default rather than null.  We should either use that or replicate it.

Reply:
Fair, will do.